tests/store/test_panel_filters.py

```diff
@@ -0,0 +1,50 @@
+from sqlalchemy.orm import Query
+
+from cg.store.api.core import Store
+from cg.store.models import Panel
+from cg.store.panel_filters import filter_panel_by_abbrev
+
+
+def test_filter_panel_by_abbrev_returns_correct_panel(store_with_panels: Store):
+    """Test finding a panel by abbreviation when the abbreviation exists."""
+
+    # GIVEN a store with multiple panels
+    num_panels = store_with_panels._get_panel_query().count()
+    assert num_panels > 0
+
+    # Select a random panel from the store
+    panel = store_with_panels._get_panel_query().first()
+    assert isinstance(panel, Panel)
+
+    # WHEN finding the panel by abbreviation
+    filtered_panel_query: Query = filter_panel_by_abbrev(
+        panels=store_with_panels._get_panel_query(),
+        abbreviation=panel.abbrev,
+    )
```

Comment:
I think if a variable is going to be used only once maybe is better to just use the function call instead, while if the function call is used several times is better to create a variable
```suggestion
    panel_query: Query = store_with_panels._get_panel_query()
    assert panel_query.count() > 0

    # GIVEN a panel from the store
    panel: Panel = panel_query.first()
    assert isinstance(panel, Panel)

    # WHEN finding the panel by abbreviation
    filtered_panel_query: Query = filter_panel_by_abbrev(
        panels=panel_query,
        abbreviation=panel.abbrev,
    )
```